tests/unittests/simulation/test_output_values.py

```diff
@@ -43,65 +36,82 @@ def test_component_and_flow_output_object() -> None:
             scenario=0,
         ): mock_variable_component,
     }
-
     opt_context.block_length.return_value = 1
+    opt_context.network = Mock()
+    opt_context.network.all_components = []
 
-    mock_problem.context = opt_context
     mock_solver = Mock()
     mock_solver.IsMip.return_value = False
+
+    mock_problem = Mock(spec=OptimizationProblem)
+    mock_problem.context = opt_context
     mock_problem.solver = mock_solver
-    output = OutputValues(mock_problem)
 
+    # ---------------------------------------------
+    # Patch extra output evaluation to return empty
+    # ---------------------------------------------
+    with patch(
+        "gems.simulation.extra_output.evaluate_extra_outputs_for_a_component",
+        return_value={},
+    ):
+        output = OutputValues(mock_problem)
+
+    # ------------------------
+    # Build expected OutputValues
+    # ------------------------
     test_output = OutputValues()
-    assert output != test_output, f"Output is equal to empty output: {output}"
+    assert (
+        output != test_output
```

Comment:
Please use `expected_output` for the reference output for the test. With output/test_output, I do not really know quickly which one is the test reference and which one is the "test output"...